packages/rocketchat-lib/server/models/Rooms.coffee

```diff
@@ -230,7 +230,7 @@ class ModelRooms extends RocketChat.models._Base
 	findByTypeContainingUsernames: (type, username, options) ->
 		query =
 			t: type
-			usernames: { $all: [].concat(username) }
+			usernames: { $contains: [].concat(username) }
```

Comment:
IMHO, username might be an array or a string. It is not a problem if it is a string, but it can be a double array if it is an array. the reason for using concat is to counter both cases.